metabase/metabase
# Remove chart rendering throttle with the app sidebar animation

Closes https://github.com/metabase/metabase/issues/24291

## Changes

It is controversial, but this PR reverts chart rendering throttle because it slowed down overall speed of rendering and customers want to have faster rendering. Unfortunately, rewriting of line/area/bar/combo charts to properly deal with performance issues is not a feasible task to do at the moment, that is why I simply revert the throttle.

In addition to that, I'm also removing the collections sidebar animation.
- it causes multiple rerenders of charts when resizing not instantly
- all other sidebars do not have any animation

## How to verify

- create and open a dashboard with multiple linea/area/bar/combo charts
- ensure it does not flash white cards between showing a loader and a chart
